src/layer/form-widgets/index.md

```diff
@@ -0,0 +1,298 @@
+
+Extra configuration of attribute forms can be done to make data collection easier and more consistent, such as using default values, conditional visibility or constraint enforcement. These topics are covered in [Attributes Form Configuration](../form-configuration/) and [Attributes Form Layout](../form-layout/).
+
+<YouTube id="jc4p1PpXj3k" />
+
+## Widget gallery
+Attributes forms can be configured using <QGISHelp ver="latest" link="user_manual/working_with_vector/vector_properties.html#edit-widgets" text="QGIS widget types" /> in the **Attributes Form** tab in the **Layer Properties**.
+
+A number of QGIS widget types can be used in <MobileAppName />, such as *Text Edit*, *Range* widget for numbers, *Date/Time* widget with calendar popup, *Value Map* and *Value Relation* to select options from a drop-down menu, or the *Attachment* widget that is used to save photos. The widget has to be compatible with the data type of a field. 
+
+Here is an overview of widgets commonly used both in <QGIS link="" text="QGIS" /> and the <MobileAppNameShort />.
+
+|QGIS widget  | Description  |<div style="width:300px">Preview in <MobileAppName /> </div> |
+|:---:|:---:|:---:|
+|Text Edit  |[Text](#text)   | ![Mergin Maps mobile app text field form](./mobile-form-text-preview.jpg "Mergin Maps mobile app text field form") |
+|QR & barcode scanner  |[Camera to scan QR and barcode](#qr-code)   | ![Mergin Maps mobile app QR code field form](./mobile-form-qrcode-preview.jpg "Mergin Maps mobile app QR code field form") |
+|Range - Editable   |[Enter a number](#numbers)   | ![Mergin Maps mobile app numeric field form](./mobile-form-number-editable-preview.jpg "Mergin Maps mobile app numeric field form")  |
+|Range - Slider  |[Select a number using the slider](#slider)   | ![Mergin Maps mobile app slider field form](./mobile-form-number-slider-preview.jpg "Mergin Maps mobile app slider field form")  |
+|Date/Time   |[Calendar with time](#date-and-time)  |![Mergin Maps mobile app date time field form](./mobile-form-datetime-preview.jpg "Mergin Maps mobile app date time field form")   | 
+|Checkbox   |[Checkbox](#checkbox)   |![Mergin Maps mobile app checkbox field form](./mobile-form-checkbox-preview.jpg "Mergin Maps mobile app checkbox field form")   |
+|Value Map   |[Drop-down menu with predefined values](#value-map) |![Mergin Maps mobile app value map field form](./mobile-form-valuemap-preview.jpg "Mergin Maps mobile app value map field form")   |
+|Value Relation   |[Drop-down menu with values from another table](#value-relation) |![Mergin Maps mobile app value relation field form](./mobile-form-value-relation-preview.jpg "Mergin Maps mobile app value relation field form")   |
+|Attachment   |[Photos from device's camera or gallery](#attachment) | ![Mergin Maps mobile app attachment field form](./mobile-form-photo-preview.jpg "Mergin Maps mobile app attachment field form")  |
+
```

Comment:
How about adding `relations`, `gallery` and `relation reference` to the table too? They could all refer to the last section `#capturing-photos-cascade-forms-1-n-relations`

Reply:
Great idea! 
`relation reference` (used in the example of attaching multiple pics to one feature) and `relations` (used in multiple records linked to one feature) added to the list - please check if this is what you've had in mind.
The links refer to directly the pages that describe the use, as the last section is basically a crossroads with links to these pages.

Reply:
Yes, great, thanks for adding those to the list. However, I'd probably split the "Relation reference" and "Gallery" to two separate rows - the gallery could be just another "Relations" QGIS type and "Relation reference" is for child features to change a parent (no matter if using the gallery mode or text mode).